salt/modules/haproxyconn.py

```diff
@@ -45,7 +53,7 @@ def _get_conn(socket='/var/run/haproxy.sock'):
     return ha_conn
 
 
-def list_servers(backend, socket='/var/run/haproxy.sock', objectify=False):
+def list_servers(backend, socket=DEFAULT_SOCKET_URL, objectify=False):
```

Comment:
If you are going to change this, please document what the default socket is so that it shows up in the sphinx docs.

With the `/var/run/haproxy.sock` in the definition, we can see what the default is in the definition in the docs.